estate/models/estate_property_custom.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class ResUser(models.Model):
```

Comment:
```suggestion
class ResUsers(models.Model):
```

Also the file name should be res_users, as it should match the inherited file name.